### PR TITLE
Match rdb and pif

### PR DESCRIPTION
**Pull request by Yanis42**
rdb.c is matched and pif.c have some non-matchings

### CODE REVIEW

File: src/emulator/pif.c
Comment (cadmic) [inline src/emulator/pif.c:1]: what are the nonmatchings here? I tried to check out your branch but it doesn't build for me:

```
src\emulator\pif.c:116: function call 'fn_80062E5C(struct Controller *, long, 
src\emulator\pif.c:116: long *)' does not match
src\emulator\pif.c:116: 'fn_80062E5C()'
```

Comment (cadmic) [reply src/emulator/pif.c:1]: pifExecuteCommand: https://decomp.me/scratch/U1mJS
pifProcessInputData: https://decomp.me/scratch/uBCpm
pifProcessOutputData: https://decomp.me/scratch/YeZGp

Comment (Yanis42) [reply src/emulator/pif.c:1]: thanks! 🎉

Comment (Yanis42) [reply src/emulator/pif.c:1]: (also I fixed the controller prototypes)

**Review by cadmic — APPROVED** [review]

**Pull request merged**